appveyor.yml

```diff
@@ -1,76 +1,42 @@
-# https://ci.appveyor.com/project/OpenWaterAnalytics/stormwater-management-model
-
+version: 2.0.{build}
+
+image:
+- Visual Studio 2013
+- Visual Studio 2015
+- Visual Studio 2017
+configuration:
+- Release
+platform:
+- x64
+environment:
+  matrix:
+  - arch: Win32
+  - arch: Win64
+  
 matrix:
   fast_finish: true
 
-environment:
-  global:
-    # Environment variables used by astropy helpers
-    PYTHON: "C:\\conda"
-    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
-
-  matrix:
-    - PYTHON_VERSION: "2.7"
-      PYTHON_ARCH: "32"
-      ARCH: "x86"
-      platform: "x86"
-    - PYTHON_VERSION: "3.4"
-      PYTHON_ARCH: "32"
-      ARCH: "x86"
-      platform: "x86"
-    - PYTHON_VERSION: "3.6"
-      PYTHON_ARCH: "32"
-      ARCH: "x86"
-      platform: "x86"
-    - PYTHON_VERSION: "2.7"
-      PYTHON_ARCH: "64"
-      ARCH: "amd64"
-      platform: "x64"
-    - PYTHON_VERSION: "3.4"
-      PYTHON_ARCH: "64"
-      ARCH: "amd64"
-      platform: "x64"
-    - PYTHON_VERSION: "3.6"
-      PYTHON_ARCH: "64"
-      ARCH: "amd64"
-      platform: "x64"
+# skip unsupported combinations
+init:
+- set _arch=
+- if "%arch%"=="Win64" ( set _arch= Win64)
+- echo %_arch%
 
-cache:
-  - "C:\\condacache\\pkgs"
+- echo %APPVEYOR_BUILD_WORKER_IMAGE%
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2013" ( set generator="Visual Studio 12 2013%_arch%" )
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" ( set generator="Visual Studio 14 2015%_arch%" )
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" ( set generator="Visual Studio 15 2017%_arch%" )
+- echo %generator%
```

Comment:
All these would need to be updated as well

Reply:
@goanpeca, The way to handle the different versions of vs is by setting the generator to use the desired visual studio. The three images that I listed are the only appveyor options.

Reply:
Yes, but you still have the other versions available preinstalled , so we should be using them

https://www.appveyor.com/docs/build-environment/#pre-installed-software

<img width="691" alt="screen shot 2017-12-08 at 14 27 37" src="https://user-images.githubusercontent.com/3627835/33781805-ff9f699a-dc23-11e7-8f06-b848f3034d8a.png">

You just need to define the env matrix to include the right ones.

Reply:
Don’t worry - I plan on doing this

Reply:
We could just use the "Visual Studio 2015" image that includes 

2008, 2010, 2013 and 2015 and ignore 2017 and simply run the environment matrix as I suggested and there is no need to use conditionals